test/integration/api/post.js

```diff
@@ -34,83 +34,105 @@ const School = bookshelf.model('School');
+      let hashtag;
 
-      await expect(
-        {url: `/api/v1/posts/tag/${name}`, method: 'GET'},
-        'to have body array length',
-        1
-      );
-    });
-  });
+      before(async () => {
+        hashtag = await post.hashtags().create(HashtagFactory.build());
+      });
 
-  describe('/api/v1/posts/school/:url_name', () => {
-    let school;
+      after(async () => {
+        await hashtag.destroy();
+      });
 
-    before(async () => {
-      school = await post.schools().create(SchoolFactory.build());
-    });
+      it('responds with hashtag posts', async () => {
+        const name = encodeURIComponent(hashtag.attributes.name);
 
-    after(async () => {
-      await post.schools().detach(school);
+        await expect(
+          {url: `/api/v1/posts/tag/${name}`, method: 'GET'},
+          'to have body array length',
+          1
+        );
+      });
     });
 
-    it('responds with school posts', async () => {
-      const name = encodeURIComponent(school.attributes.url_name);
+    describe('/api/v1/posts/school/:url_name', () => {
+      let school;
 
-      await expect(
-        {url: `/api/v1/posts/school/${name}`, method: 'GET'},
-        'to have body array length',
-        1
-      );
-    });
-  });
+      before(async () => {
+        school = await post.schools().create(SchoolFactory.build());
+      });
 
-  describe('/api/v1/posts/geotag/:url_name', () => {
-    let geotag;
+      after(async () => {
+        await post.schools().detach(school);
+      });
 
-    before(async () => {
-      geotag = await post.geotags().create(GeotagFactory.build());
+      it('responds with school posts', async () => {
+        const name = encodeURIComponent(school.attributes.url_name);
+
+        await expect(
+          {url: `/api/v1/posts/school/${name}`, method: 'GET'},
+          'to have body array length',
+          1
+        );
+      });
     });
 
-    after(async () => {
-      await post.geotags().detach(geotag);
+    describe('/api/v1/posts/geotag/:url_name', () => {
+      let geotag;
+
+      before(async () => {
+        geotag = await post.geotags().create(GeotagFactory.build());
+      });
+
+      after(async () => {
+        await post.geotags().detach(geotag);
+      });
+
+      it('responds with geotag posts', async () => {
+        const name = encodeURIComponent(geotag.attributes.url_name);
+
+        await expect(
+          {url: `/api/v1/posts/geotag/${name}`, method: 'GET'},
+          'to have body array length',
+          1
+        );
+      });
     });
 
-    it('responds with geotag posts', async () => {
-      const name = encodeURIComponent(geotag.attributes.url_name);
+    describe('/api/v1/posts/liked/:name', () => {
+      let postHashtagLike;
+
+      before(async () => {
+        postHashtagLike = await new Post(PostFactory.build({type: 'hashtag_like'})).save(null, {method: 'insert'});
+      });
+
+      after(async () => {
+        postHashtagLike.destroy();
```

Comment:
add either `return` or `await` here. otherwise, promise will be executed in background, asynchronously and without error-checks